src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -367,11 +379,23 @@ trait SendsToAmplitudeWithEnvironment extends SendsToAmplitude {
 
     ("user_properties" ->
       ("channel" -> meta.normalizedChannel) ~
-        ("app_build_id" -> meta.appBuildId) ~
-        ("locale" -> meta.`environment.settings`.map(_.locale)) ~
-        ("is_default_browser" -> meta.`environment.settings`.map(_.isDefaultBrowser)) ~
-        ("experiments" -> experimentsArray)) ~
-      ("user_id" -> getClientId)
+      ("sample_id" -> meta.sampleId) ~
+      ("app_build_id" -> meta.appBuildId) ~
+      ("app_name" -> meta.appName) ~
+      ("app_version" -> meta.appVersion) ~
+      ("locale" -> meta.`environment.settings`.map(_.locale)) ~
+      ("is_default_browser" -> meta.`environment.settings`.map(_.isDefaultBrowser)) ~
+      ("country" -> meta.geoCountry) ~
+      ("os" -> meta.os) ~
+      ("os_version" -> meta.`environment.system`.map(_.os.name)) ~
+      ("env_build_arch" -> meta.`environment.build`.map(_.architecture)) ~
+      ("is_wow64" -> meta.`environment.system`.map(_.isWow64)) ~
+      ("memory_mb" -> meta.`environment.system`.map(_.memoryMB)) ~
```

Comment:
I think for some of the less general-purpose fields here (env_build_arch, is_wow64) it might make sense to add them to the schema as user properties instead of adding them for every job that uses this code

Reply:
I think we'd have to add them to every event in the schema definition if we don't do it here. This seems like a simpler solution.

Reply:
Well, since there aren't any other jobs running regularly right now I think that's fine but 1. we should consider adding top-level user properties that apply to all events in the schema/group as a follow-on bug or 2. seriously consider moving away from json definitions and moving toward a more flexible dsl